### ISSUE TITLE
[Feature Request] Choosing different styles for tasks 

### ISSUE THREAD

**Issue by doschroeder** [ivan-lednev/obsidian-day-planner#359]
**Fantastic plugin! I'm very grateful!**

**Suggestion:** It would be amazing to have more styling options for my scheduled tasks. Right now, the only customization available is through heading levels, but I'd prefer to format my tasks like this:

```

>[!Daily tasks]
> - [ ] 8:00 - 8:30 Make coffee 
> - [ ] 8:30 - 9:00 Review emails 

```

Could there be a feature to include code in the heading text and offer more flexibility in task formatting?

**Comment by ivan-lednev** [comment]
Hello, @doschroeder,

I think those tasks should already be visible in the planner, since it collects all the tasks in daily notes by default. The heading from the config serves only as a destination point for tasks created with drag-and-drop.

**Comment by doschroeder** [comment]
Hello @ivan-lednev,

many thanks for the reply. No, unfortunately not. If I am writing >[!Daily tasks] then it inserts #>[!Daily tasks] 

Or am I missing something?

**Comment by ivan-lednev** [comment]
Oh, I see what you meant now. For now, if you want to use a custom section, you can create a daily note template with that `> [!Daily tasks]`

**Comment by doschroeder** [comment]
Yes, this is what it did. But when I insert a new time block in the week planer, it adds a new section. Many thanks for your great work! =)
